languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/style.xml

```diff
@@ -6147,9 +6147,9 @@ USA
                 <example correction="Familienfotografien">Ihr gelangen immer wieder wundervolle <marker>Familienphotographien</marker>.</example>
             </rule>
         </rulegroup>
-        <rulegroup id="TEST_F_ANSTATT_PH" name="Worte mit Ph anstatt F" tone_tags="formal">
+        <rulegroup id="F_ANSTATT_PH_2" name="Worte mit Ph anstatt F" tone_tags="formal">
```

Comment:
nur dran denken, dass die regel jetzt wieder für alle anschlägt, die sie mal disabled haben

Reply:
Danke, ja, aber ich finde, sie sollte nicht mehr den Namen TEST haben, wenn sie schon längst in Produktion ist ... Oder?

Reply:
ja, denke ich auch. nur in bezug auf die performance könnte es vermehrt dann zu disables kommen, nicht, dass man sich wundert oder so